x/leverage/keeper/interest.go

```diff
@@ -133,11 +133,14 @@ func (k Keeper) AccrueAllInterest(ctx sdk.Context) error {
 			interestAccrued.TruncateInt(),
```

Comment:
here we truncate (take a floor). Potentially `totalInterest` won't much accumulated

Reply:
This line specifically is for logging, so it's probably ok, but there is still a danger of interest per block being less than one base token, in which case it must be treated as zero.